packages/daft/daft.0.0.4/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/daft"
+bug-reports: "https://github.com/UnixJunkie/daft/issues"
+dev-repo: "git+https://github.com/UnixJunkie/daft.git"
+license: "CeCILL-C"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "5.0.0"}
+  "dune" {>= "1.1"}
+  "dolog" {>= "4.0.0"}
+  "batteries"
+  "fileutils"
+  "zmq" {>= "5.0.0" & < "5.1.4"}
```

Comment:
This upper-bound seems rather weird. Is this really not working with 5.1.4 onward?
```suggestion
  "zmq" {>= "5.0.0"}
```